integration-tests/README.md

```diff
@@ -0,0 +1,31 @@
+# ink! code examples
+
+This folder contains a set of example contracts for ink!.
```

Comment:
We should mention, as you do in the PR description, that the main purpose of the examples in this folder is now for testing `ink` end to end as part of the development/CI process, and as such this folder contains examples which are testing specific features, which are distinct from real world examples.